parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/BytePacker.java

```diff
@@ -105,4 +116,26 @@ public void unpack8Values(final byte[] input, final int inPos, final int[] outpu
   public void unpack32Values(byte[] input, int inPos, int[] output, int outPos) {
     unpack32Values(ByteBuffer.wrap(input), inPos, output, outPos);
   }
+
+  /**
+   * unpack bitWidth bytes from input at inPos into {unpackCount} values in output at outPos using Java Vector API.
+   * @param input the input bytes
+   * @param inPos where to read from in input
+   * @param output the output values
+   * @param outPos where to write to in output
+   */
+  public void unpackValuesVector(final byte[] input, final int inPos, final int[] output, final int outPos) {
+
+  }
+
+  /**
+   * unpack bitWidth bytes from input at inPos into {unpackCount} values in output at outPos using Java Vector API.
+   * @param input the input bytes
+   * @param inPos where to read from in input
+   * @param output the output values
+   * @param outPos where to write to in output
+   */
+  public void unpackValuesVector(final ByteBuffer input, final int inPos, final int[] output, final int outPos) {
+
```

Comment:
It is better to throw here and above?

Reply:
@wgtmac I don't understand it, not throw exception here. Can you you describe in detail ？ thanks.

Reply:
I mean why is it an empty function? Shouldn't it throw an unimplemented exception?